src/lib/y2storage/blk_device.rb

```diff
@@ -317,6 +317,16 @@ def delete_filesystem
       remove_descendants
     end
 
+    # Mount point of the filesystem
+    #
+    # @return [Y2Storage::MountPoint, nil] nil if the device is not formatted or its
+    #   filesystem has not mount point.
+    def mount_point
+      return nil unless formatted?
+
+      filesystem.mount_point
```

Comment:
nice shortcut :+1: . Maybe search rest of existing code and use it where it make sense?

Reply:
Well, this PR already includes a lot changes. Due to this is for GA, I would avoid to add more changes here and there. I would prefer to do this "refac" in a separate PR.